packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -228,6 +295,67 @@ describe.each([['petstore.oas2.json'], ['petstore.oas3.json']])('server %s', fil
     }
   });
 
+  describe('server validation: given __server query param', () => {
```

Comment:
According to Phil (or at least, if I understood correctly), these should be two nested describe blocks